solace-deployment.yml

```diff
@@ -1268,6 +1272,7 @@ instance_groups:
           longDescription: Solace PubSub+ Message Broker for real-time, multi-protocol data distribution
           documentationUrl: http://docs.pivotal.io/partners/solace-pubsub/
           supportUrl: http://dev.solace.com
+          shareable: true
```

Comment:
for completeness, we should do the same for that value since it is a user configurable field. Parse it in bosh-lite-testing.sh and use a variable here